src/helpers/types.ts

```diff
@@ -5,6 +5,10 @@ export interface Query extends core.Query { }
 
 export interface Params extends core.ParamsDictionary { }
 
-export interface Request<ReqBody = any, ReqQuery = Query, URLParams extends Params = core.ParamsDictionary>
+export interface Request<
+ReqBody = any,
+ReqQuery = Query,
+URLParams extends Params = core.ParamsDictionary,
+>
 
-extends express.Request<URLParams, any, ReqBody, ReqQuery> {}
+  extends express.Request<URLParams, any, ReqBody, ReqQuery> {}
```

Comment:
I prefer to put this extends on the same line with `>`, so we know that it belongs to `Request`

Reply:
Because the first time I read it, I don't know why there's an `extend` statement here._.

Reply:
> btw saran aja ini mah y. sekalian sharing sdkit ilmu yg pernah di dapet di techtalk2 lapak sebelah terkait git proses ini smile
> 
> jadi better untuk yang big commit gini mending di pisah ke beberapa branch dengan related overview/ issue/ dsb. dalam kata lain branch nya jadi banyak lah ya. laughing tapi ini jadi lebih baik karena pada proses reviewer akan lebih detil dan teliti jikalau misalkan ada code yg memang perlu di perbaiki. beda dengan panjang seperti ini biasanya udah lelah di awal dan masih page atas. ( mending kalo ngerti code nya, coba kalo engga ya pasti orang langsung next #gitukatanya) laughing. karna ini masih proses development ga papa branch nya jadi di pisah2 per task/ per function/ per prosess. beda halnya jika branch nya yg mau up production big commit ini ga terlalu di permasalahkan karena hal2 yang di review hanya minor2 saja. catatan sya : katanya big commit bukan selalu commit msg yg banyak di suatu branch. tapi bisa juga contoh kasus nya seperti ini commit nya mah 2 tapi line code nya > 1000 line code jika di gabung2 per pagenya laughing
> 
> sekian dan trimakasih. +1

ya tetapi untuk case ini memang pas sebelumnya ada request linter config airbnb, cuman waktu itu masih terkendala problem, nah mumpung lagi low task, langsung run lint:fix karena memang di CI/CD nya pun aku pasang check eslint nya, jadi untuk case ini blm bisa per kasus, begitu mangs :smile:

Reply:
This extend is still connected, so it's written like that